### ISSUE TITLE
Profile card rendering looks surprising in Safari

### ISSUE THREAD

**Issue by ojacobson** [wlonk/wheretofind.me#103]
In the attached image, the card has a white stripe at the bottom and right-hand edges, overlapping the signal strength indicator and causing the label and icon to be positioned off of the vertical center of the card. (Safari: Version 12.0.2 (13606.3.4.1.4))

<img width="319" alt="screen shot 2019-02-12 at 10 58 03 pm" src="https://user-images.githubusercontent.com/160985/52685976-c0fced80-2f19-11e9-825e-1be12bcbd188.png">

Possible see also: #101

**Comment by ojacobson** [comment]
This anomaly is also present on the edit page:

<img width="695" alt="screen shot 2019-02-12 at 11 00 25 pm" src="https://user-images.githubusercontent.com/160985/52686039-03bec580-2f1a-11e9-9d5d-1633643dfca5.png">

**Comment by wlonk** [comment]
Can't reproduce with Safari 12.0.3. Is it possible it's something cached or a plugin?

**Comment by ojacobson** [comment]
Doesn't appear so. The only extensions I have enabled are uBlock Origin and 1Password. For a lark, I disabled both and reloaded the page; I got the same anomalous rendering. I also tried an incognito tab to avoid cache or cookie issues (same result).

I dug into the DOM a bit, and the inner grey area is .card-body, while the outer light area is .card. The DOM inspector shows a 40 pixel-ish difference in their sizes, in both dimensions.
